src/components/Book.jsx

```diff
@@ -1,13 +1,19 @@
+/* eslint react/prop-types: 0 */
```

Comment:
- [ ] Kindly avoid disabling liners and fix them because linters help you write clean code follow best practise.

Reply:
Thankyou @omarsalem7 I have fixed this issue.

Reply:
- Well resolved 👍🏽